packages/deployment/app/config.yml

```diff
@@ -119,16 +127,6 @@ default:
   es:
     name: es5vpc
     elasticSearchMapping: 8
-    version: 5.3
-    instanceCount: 1
-    instanceType: t2.small.elasticsearch
-    volumeSize: 35
```

Comment:
Perhaps I'm misreading, but `es.VolumeSize` is still required in the DB template, and it has no fallback default value. So don't we still need that as part of the default config?

Reply:
Nevermind, I see that this was moved to the DB config file